### ISSUE TITLE
[BUG] [Linter] H025 False positive when trans tag contains one single quote 

### ISSUE THREAD

**Issue by a-edakin** [Riverside-Healthcare/djLint#447]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues) I hope I checked all issues related to H025 if it's duplicate sorry:(
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: I don't think it's relevent because I could reproduce it on any OS 
 - Python Version: 3.10.2
 - djLint Version: 1.19.4
 - template language: django

## Issue
False positive H025 issue if HTML's tag attribute contains `{% trans "some text with ONE single ' quote" %}`.

But if I would use second quote then everything would be fine `{% trans "some text with TWO single '' quote" %}`. 

## How To Reproduce

HTML file to reproduce:
```html
<button title="{% trans "text with ONE single ' quote" %}">
</button>

<a title="{% trans "text with ONE single ' quote" %}">
</a>
``` 

Command: `djlint -`

Console output:
```shell
templates/test_tag_trans.html
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
H025 1:0 Tag seems to be an orphan. <button title="{% tr
H025 5:0 Tag seems to be an orphan. </a>

Linted 34 files, found 2 errors.
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]
```toml
[tool.djlint]
profile="django"
indent = 4
max_line_length=120
blank_line_before_tag="include,for,block"
blank_line_after_tag="load,include,endfor,endblock"
ignore="H021,H022,H006"
files = ["templates"]
```

**Comment by christopherpickering** [comment]
Attribute parsing is gonna kill me lol. I'll put out a fix shortly. I'm working hard on an update that will get rid of most regex 😅

**Comment by a-edakin** [comment]
To fix this issue I just moved trans tag out of attributes 

```html
{% trans "text with ONE single ' quote" as tanslated_text %}
<button title="{{ tanslated_text }}">
</button>
```

**Comment by SebastiaanZ** [comment]
You've probably already know this, but it's not limited to the `trans` tag:

```diff
     <p>
         <some-tag
-                    onclick="window.location.href = '{% url 'sso:logout' %}'">
+                    onclick="window.location.href = '{% url 'sso:logout' %}">
         </some-tag>
     </p>
```

Although my guess is that you use a generic, not tag/filter specific, method of counting nested quotation marks.

**Comment by christopherpickering** [comment]
@SebastiaanZ @a-edakin thanks! I should have a release out shortly that covers both cases.
